socorro/external/es/crashstorage.py

```diff
@@ -37,6 +37,22 @@ def __init__(self, config):
         ]
 
 
+# Valid Elasticsearch keys contain one or more ascii alphanumeric characters, underscore, and hyphen
+# and that's it.
+VALID_KEY = re.compile(r'^[a-zA-Z0-9_-]+$')
```

Comment:
This rule is based on the rule in this comment:

https://bugzilla.mozilla.org/show_bug.cgi?id=1382727#c6

The rule in that comment looks for "bad characters". This is the inverse of that where it makes sure the key only contains good characters.